functions/.env

```diff
@@ -1,2 +1,7 @@
 POSTFINANCE_EMAIL_USER=test@socialincome.org
 POSTFINANCE_EMAIL_PASSWORD=password
+
+TWILIO_SID=ACac5dfe59a0af841c6b771835f331fabf
```

Comment:
Are those the prod ones 😆 ?
Let's reset them in twilio and use new ones?

Reply:
no only test 🙈 but I thought I added this file to .gitignore. --> doesn't seem the case 😭